content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,151 @@
+Traditionally it was indistinguishable from `EventStorming`, however its evolution has lead to Process Modelling becoming its own phase in the `EventStorming` methodology.
+
+While the formal method is best used for identifying large complex high level processes, the general principals and grammar of Process Modelling have been adopted successfully to discover or validate workflows and pipelines at much smaller scales and through existing systems.
+
+The output from these workshops can be used to discover, validate or design a system that supports an end to end business process. The information can be used as an algorithmic model of the problem space, and other forms of design can be derived from it; such as an Event + Domain Driven microservice architecture, a simple Pipeline or even design a `CQRS` and Event Sourcing pattern. Typically there will be a follow on exercise to rationalize the information and transform it from an algorithmic model to a more suitable architectural model.
+
+#### Sample Agenda & Prompts - 
+1. Arrange the board:
+     - Label the board to identify a name for the process being explored
+     - From the main storming board, identify a selection of key events that describe an end to end process within the larger system and copy them to another board. 
+       - Preconditions: Identify events which need to have happened before this process can begin and put them in a grayed area on the left with the title Preconditions
+       - Postconditions: Identify at least one suitable event which describes the optimal outcome of the process
+       - Arrange any remaining events in a chronological order from left to right between the preconditions and optimal outcomes. Do not worry about accuracy, as the events are narrated, be prepared to change, add or remove events
+     - Add a grammar guide as shown  ![This is the placeholder for process modelling grammar](/images/practices/event-storming-process-modelling/process-modelling-grammar.png)
+
+1. First Narration: Attempt to narrate a `golden path` through the board, describing the process of events as they happen from left to right. Adjust and add grammar as necessary to reach a consensus of the process.
+    - Roles: 
+       - There are typically 3 roles; one Narrater, one Scribe and the remaining participants are The Audience
+         - A Narrator reads through the process from left to right
+         - The Audience listens to the narrator and assists in discovering the process
+         - A Scribe records the process on the board
+       - Any participant can take on any of these roles and it should be encouraged for roles to rotate
+
+    - Questions: If a question cannot be answered straight away then leave it on the board and return to it later.
+    - Grammar: Add grammar appropriately but do not get bogged down by it
```

Comment:
Ohhhh! Here it is. What about moving this up? Isn't it more a "Legend" than a "Grammar"?

Would be great to have a visual of the colours, matching the definitions.

The parallel numbers of this ordered list can be confusing with the main steps of the practice. What about using  unordered bullet points or a. b. c. ... ?